ui/selectors/selectors.js

```diff
@@ -2529,6 +2529,11 @@ export function getSnapsInstallPrivacyWarningShown(state) {
   return snapsInstallPrivacyWarningShown;
 }
 
+export function getConfirmationAdvancedDetailsOpen(state) {
```

Comment:
Can we avoid adding selectors to this legacy huge file and instead use our more targeted `ui/pages/confirmations/selectors/confirm.ts`?